examples/reliable_conversation/src/tasks/task_registry.py

```
@@ -0,0 +1,18 @@
+"""
+Task registry for RCM quality control tasks.
+Registers all tasks with the app instance.
+"""
+
+from mcp_agent.app import MCPApp
```

Pylint (3.3.7)

[error] Unable to import 'mcp_agent.app'

(E0401)